packages/components/src/components/@shared/_mixins.scss

```diff
@@ -1,3 +1 @@
-@function to-rem($size) {
-	@return calc(#{$size}rem / var(--kolibri-root-font-size, 16));
-}
+@use '../../../to-rem' as *;
```

Comment:
Die SCSS to-rem Methode sollte weiterhin innerhalb unserer src verzeichnisses liege. wir sollten sie erstmal da lassen wo sie war.

Aber im root-Verszeichnis von components package könnten wir doch eine index.scss hinterlegen, die die to-rem methode aus source importiert und exoertiert. So wie man das in typescript mit index.ts barrel files macht.